mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -303,7 +303,7 @@ class LogisticRegression @Since("1.2.0") (
 
     val (coefficients, intercept, objectiveHistory) = {
       if (numInvalid != 0) {
-        val msg = s"Classification labels should be in {0 to ${numClasses - 1} " +
+        val msg = s"Classification labels should be in {0 to ${numClasses - 1}}. " +
```

Comment:
While we're here, use [ and ] ? these mean "inclusive" but curly braces don't. Or just write "between 0 and ${numClasses - 1}"